pyds8k/test/test_resources/test_ds8k/test_volume.py

```diff
@@ -469,3 +476,91 @@ def test_create_volume_type_error(self):
             self.system.create_volume('name', '10', 'testpool_0',
                                       types.DS8K_VOLUME_TYPE_FB,
                                       tp='fake_tp')
+
+    @httpretty.activate
+    def test_create_flashcopy(self):
```

Comment:
I think these new tests belong in a new `test_flashcopies.py` file

Reply:
done